docs/snippets/movidius-programming-guide.cpp

```diff
@@ -1,36 +1,36 @@
 #include <inference_engine.hpp>
 
 int main() {
-InferenceEngine::Core core;
-int numRequests = 42;
-int i = 1;
-auto network = core.ReadNetwork("sample.xml");
-auto executable_network = core.LoadNetwork(network, "CPU");
-//! [part0]
-struct Request {
-    InferenceEngine::InferRequest inferRequest;
-    int frameidx;
-};
-//! [part0]
+    InferenceEngine::Core core;
+    int numRequests = 42;
+    int i = 1;
+    auto network = core.ReadNetwork("sample.xml");
+    auto executable_network = core.LoadNetwork(network, "CPU");
+    //! [part0]
+    struct Request {
+        InferenceEngine::InferRequest inferRequest;
+        int frameidx;
+    };
+    //! [part0]
 
-//! [part1]
-// numRequests is the number of frames (max size, equal to the number of VPUs in use)
-std::vector<Request> request(numRequests);
-//! [part1]
+    //! [part1]
+    // numRequests is the number of frames (max size, equal to the number of VPUs in use)
+    std::vector<Request> request(numRequests);
+    //! [part1]
```

Comment:
in this case all snippets in docs are shifted by 4 spaces..

Reply:
Yes good point.Fixed indentation for snippets